stake-pool/program/src/processor.rs

```diff
@@ -170,25 +288,39 @@ impl Processor {
         let account_info_iter = &mut accounts.iter();
         let stake_pool_info = next_account_info(account_info_iter)?;
         let owner_info = next_account_info(account_info_iter)?;
```

Comment:
There are pros and cons here. Enforcing owner signature does help with consistency, but there is also a case when stake pool initialized for the particular owner by somebody else. But from the other hand this scenario is less likely because owner signature is required anyway to add new validator stake accounts to the pool...
So I'm also leaning toward requiring owner signature on initialization.

Reply:
That's a good point -- I'll let you decide what's best

Reply:
Let's keep it as it is now and I'm adding a task for the next PR to change it to require owner to sign Initialize instruction.